core/src/main/java/de/danoeh/antennapod/core/service/playback/RemotePSMP.java

```diff
@@ -546,13 +546,13 @@ public void endPlayback(boolean wasSkipped, boolean switchingPlayers) {
         } catch (TransientNetworkDisconnectionException | NoConnectionException e) {
             Log.e(TAG, "Could not determine if media is playing", e);
         }
-        if (wasSkipped) {
-            try {
-                castMgr.stop();
-            } catch (CastException | TransientNetworkDisconnectionException | NoConnectionException e) {
-                Log.e(TAG, "Could not stop remote playback when skipping", e);
-            }
-        }
+//        if (wasSkipped) {
```

Comment:
I've seen this in a couple of commits (commenting out code).

Will this code need to come back?  If so, why? what's it currently break?  If not, why not just delete it?

In the last few commits there were some other things that were suspiciously commented out. Same question for them.